src/linearmap.rs

```diff
@@ -318,10 +373,29 @@ mod tests {
         )
     }
 
+    #[test]
+    fn virtual_address_range_ttbr1() {
+        const GIB_512: isize = 512 * 1024 * 1024 * 1024;
+        // Map the 512 GiB region at the top of virtual address space to the bottom of physical
+        // address space.
+        let translation = LinearTranslation::new(GIB_512);
+
+        // The first page in the region covered by TTBR1.
+        assert_eq!(
+            translation.virtual_to_physical(VirtualAddress(0xffffff8000000000)),
```

Comment:
Could we add underscores to make these more readable - 0xffff_ff80_0000_0000? Or compute it some other way, so it's obvious? -GIB_512, perhaps.

Reply:
Done.